project/threads/models.py

```diff
@@ -74,6 +85,9 @@ class Thread(models.Model):
     author = models.ForeignKey(
         Profile, default=None, null=True, on_delete=models.PROTECT
     )
+    category = models.ForeignKey(
+        Category, default=None, null=True, on_delete=models.PROTECT
```

Comment:
In order for this line to work, you will need to import the `Category` model from its current location in your branch. For example, add this to the imports section of `threads/models.py`

```py
from api.models import Category
```

Keep in mind that we aren't moving the `Category` model as part of this pull request, so it will remain in the `api` app for the timebeing. We have a separate issue to move the `Category` model to a new `categories` app #1040